src/core_modules/capture-core/components/LoadingMasks/LoadingMaskForButton.component.js

```diff
@@ -1,15 +1,7 @@
 // @flow
 import React from 'react';
-import { LoadingMask } from './LoadingMask.component';
+import { CircularLoader } from '@dhis2/ui';
 
-type Props = {
-
-};
-
-export const LoadingMaskForButton = (props: Props) => (
-    // $FlowFixMe[cannot-spread-inexact] automated comment
-    <LoadingMask
-        size={24}
-        {...props}
-    />
+export const LoadingMaskForButton = () => (
```

Comment:
NIT: Since it's no longer `LoadingMask` do you think we should rename to `CircularLoaderForButton`

Reply:
Hey,

The components in the folder `LoadingMasks` use`CircularLoader` + different other elements in their `JSX`. That is why I prefer the term `LoadingMasks...` as a generic namespace for whatever we want to render,  not only the `CircularLoader`. 

Thank you for your feedback!